.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: cpp
```

Comment:
Just asking, but isn't this c?

Reply:
gtest is used internal and it requires C.
I have no knowledge specifying both C and C++ in this format.